ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/integration/casework-regression/create-case.spec.ts

```diff
@@ -1,68 +1,134 @@
+            .hasTrustSummaryDetails("Ashton West End Primary Academy")
+            .clickAddConcernButton()
+            .hasValidationError("Select concern type")
+            .hasValidationError("Select risk rating")
+            .hasValidationError("Select means of referral");
+
+        cy.reload();
         
-        editCaseManagementPage.hasNoConcernType("Safeguarding");
-        
-		    cy.selectConcernType();
-		    cy.selectRiskToTrust();
-		    cy.selectTerritory();
+        Logger.Log("Create a valid concern");
+        createConcernPage
+            .withConcernType("Financial")
+            .withSubConcernType("Financial: Deficit")
+            .withRating("Red-Amber")
+            .withMeansOfRefferal("External")
+            .clickAddConcernButton();
 
-        Logger.Log("Validating concern details");
 
-        caseMangementPage
-            .createCase()
-            .hasValidationError("Issue is required");
+        Logger.Log("Check Concern details are correctly populated");
+        createConcernPage
+            .hasTrustSummaryDetails("Ashton West End Primary Academy")
+            .hasConcernType("Financial: Deficit")
+            .clickNextStepButton();
 
-        editCaseManagementPage.withIssue("This is an issue");
-        cy.waitForJavascript();
 
-        editCaseManagementPage.withCaseHistoryExceedingLimit();
+        Logger.Log("Check unpopulated risk to trust throws validation error");
+        addDetailsPage
+            .clickNextStepButton()
+            .hasValidationError("Select risk rating");
 
-        caseMangementPage
-            .createCase()
+        
+        cy.reload();
+        
+        Logger.Log("Populate risk to trust");
+        addDetailsPage
+            .withRating("Red-Plus")
+            .clickNextStepButton();
+
+
+        Logger.Log("Check Trust, concern and risk to trust details are correctly populated");
+        addTerritoryPage
+            .hasTrustSummaryDetails("Ashton West End Primary Academy")
+            .hasConcernType("Financial: Deficit")
+            .hasRiskToTrust(0,"Red Plus")
+
+        Logger.Log("Check unpopulated territory throws validation error");
+        addTerritoryPage
+            .clickNextStepButton()
+            .hasValidationError("An SFSO Territory must be selected");
+
+
+        Logger.Log("Populate territory");
+        addTerritoryPage
+            .withTerritory("North and UTC - North East")
+            .clickNextStepButton();
+
+        Logger.Log("Check Trust, concern, risk to trust details and territory are correctly populated");
+        addConcernDetailsPage
+            .hasTrustSummaryDetails("Ashton West End Primary Academy")
+            .hasConcernType("Financial: Deficit")
+            .hasRiskToTrust(0,"Red Plus")
+            .hasTerritory("North and UTC - North East");
+
+        //
+
+        Logger.Log("Validate unpopulated concern details");
+        addConcernDetailsPage
+            .withExceedingTextLimit("issue", 2001)
```

Comment:
QUERY
could we still do this idea, but move the calling of the function into the page object

so you would have

withIssueExceedingLimit which calls withExceedingTextLimit("issue", 2001)